### ISSUE TITLE
Does plugins installation break jellyfinserver on TrueNAS Core?

### ISSUE THREAD

**Issue by julben** [Thefrank/jellyfin-server-freebsd#17]
Hi,

I have been trying to setup and install Jellyfin on TrueNAS Core (12.3-RELEASE) a couple of times, and, every single time I installed a few plugins and rebooted the jail, Jellyfin refuses to work.

Are you aware of such an issue?

**Comment by Thefrank** [comment]
I am not aware of any issues with the TrueNAS plugin system causing issues with jellyfinserver but ideally you want one application/service per jail.

Does jellyfinserver work fine before trying to install TrueNAS plugins on top of it? What errors are produced in the logs after you restart the jail?

**Comment by julben** [comment]
I started a new jail from scratch (I have migrated my jails to another pool since the time I wrote the previous message), and everything seems fine.

The problem is nowhere to be seen. For the record, the thing was not that Jellyfin was not starting anymore, it was just taking a (very) long time to start (~1h). My working theory is that, the original pool the jail was on, was under heavy load at the time I was tinkering with Jellyfin, so, it was slowed down considerably. It is not the case anymore, so the issue does not show up.

I think it is safe to say that there was no issue at all with your package from the beginning. It was just me not noticing the obvious (pool/dataset settings, I/O/ load of the pool, etc.), and close this issue without worrying. ;)
